contracts/bonding/IBondingVotes.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts-upgradeable/interfaces/IERC6372Upgradeable.sol";
+
+import "../treasury/IVotes.sol";
+
+/**
+ * @title Interface for BondingVotes
+ */
+interface IBondingVotes is IERC6372Upgradeable, IVotes {
```

Comment:
Self-reminder: `IERC6372Upgradeable` is redundant here since it's included in `IVotes`. Remove